src/widgets/allergies/allergy-card-level-three.component.tsx

```diff
@@ -0,0 +1,164 @@
+import React from "react";
+import { match } from "react-router";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import { getPatientAllergyByPatientUuid } from "./allergy-intolerance.resource";
+import { useCurrentPatient } from "@openmrs/esm-api";
+import styles from "./allergy-card-level-three.css";
+import dayjs from "dayjs";
+import SummaryCard from "../cards/summary-card.component";
+
+export function AllergyCardLevelThree(props: AllergyCardLevelThreeProps) {
+  const [allergy, setAllergy] = React.useState(null);
+  const [isLoadingPatient, patient, patientUuid] = useCurrentPatient();
+
+  React.useEffect(() => {
+    if (!isLoadingPatient && patient) {
+      const abortController = new AbortController();
+
+      getPatientAllergyByPatientUuid(
+        patientUuid,
+        { allergyUuid: props.match.params["allergyUuid"] },
+        abortController
+      )
+        .then(allergy => setAllergy(allergy.data))
+        .catch(createErrorHandler());
+
+      return () => abortController.abort();
+    }
+  }, [isLoadingPatient, patient]);
+
+  function displayAllergy() {
+    return (
+      <SummaryCard
```

Comment:
What happens if there are no allergies to display?

Reply:
Could this situation arise? This widget would only display after you've clicked on an existing allergy.

Reply:
ooh right, it's alright then.